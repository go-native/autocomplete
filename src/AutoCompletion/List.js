import React, { Component } from 'react';

class List extends Component {
    render() {
        return (
            <ul className="list-group">
                {this.props.children}
            </ul>
        );
    }
}

export default List;