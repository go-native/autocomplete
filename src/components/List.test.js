import React from 'react';
import renderer from 'react-test-renderer';
import List from './List';

describe('<AutoCompletion />', () => {
  it('renders children', () => {
    const tree = renderer
      .create(
        <List>
          <div></div>
        </List>,
    )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
