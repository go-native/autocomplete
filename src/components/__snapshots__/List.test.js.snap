// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AutoCompletion /> renders children 1`] = `
<ul
  className="list-group"
>
  <div />
</ul>
`;
